switchboard/conditions.py

```diff
@@ -36,44 +52,54 @@ def set_values(self, name):
         if name and not self.label:
             self.label = titlize(name)
 
-    def is_active(self, condition, value):
-        return condition == value
+    def is_active(self, value, actual_value):
+        return value == actual_value
 
     def validate(self, data):
         value = data.get(self.name)
         if value:
             value = self.clean(value)
-            assert isinstance(value, basestring), 'clean methods must return strings'
+            is_string = isinstance(value, basestring)
+            assert is_string, 'clean methods must return strings'
         return value
 
-    def clean(self, value):
+    def clean(self, value):  # pragma: nocover
```

Comment:
Per my comment above... I think 100% code coverage is more accurate than 98%, when the remaining 2% are class definitions that consist entirely of `pass` or functions that exist just to define an API (e.g., subclasses are expected to implement the real logic).